books_project/books_app/models.py

```diff
@@ -1,93 +1,91 @@
-from django.db import models
-from django.core.exceptions import ValidationError
-from django.core.validators import RegexValidator
+import re
+from marshmallow import ValidationError
+from sqlalchemy import DateTime, Table, Column, Integer, ForeignKey, String, create_engine, desc, Table, Date, func, Enum
+from sqlalchemy.ext.declarative import declarative_base
+from enum import Enum as MyEnum
+from sqlalchemy.orm import relationship, sessionmaker
 
+Base = declarative_base()
```

Comment:
since all models here inherit from "Base", and they all have `id`, `created` and `updated`. can you think of a way so we don't have to define these columns in each model? maybe make them inherited from a parent class?